aviary/subsystems/propulsion/hamilton_standard.py

```diff
@@ -572,14 +560,14 @@ def initialize(self):
         self.options.declare(
             'aviary_options', types=AviaryValues,
             desc='collection of Aircraft/Mission specific options')
-        self.options.declare('num_nodes', types=int)
+        self.options.declare('num_nodes', default=1, types=int)
```

Comment:
There might have been a couple other places that also needed the default value

Reply:
Great point Carl, thanks! Modified in other places as well